### DISCUSSION TITLE
Can't get syntax for .mignore file to work

### DISCUSSION THREAD

**Discussion by theboyler** [samschott/maestral#818] [Q&A]
I'm trying to configure .mignore to exclude certain folder from syncing, so they remain on my laptop, but not on dropbox.

My Dropbox teams account is located at /Users/johnnyappleseed/Appleseed Dropbox/ and this is the same location given in Maestral's settings and this is where the .mignore files is. 

(FYI The actual dropbox location I save to is /Users/johnnyappleseed/Appleseed Dropbox/Johnny Appleseed/). 

The folder I want to ignore is /Users/johnnyappleseed/Appleseed Dropbox/Johnny Appleseed/NoSync

However, when I try to find the syntax which will exclude it, nothing seems to work. I've tried:

**NoSync
/Users/johnnyappleseed/Appleseed\ Dropbox/Johnny\ Appleseed/NoSync
Users/johnnyappleseed/Appleseed\ Dropbox/Johnny\ Appleseed/NoSync
Johnny\ Appleseed/NoSync

Can some kind soul tell me a) what the correct syntax is and b) is the .mignore file in the right place etc?

**Comment by samschott** [comment]
@theboyler, the location of the `.mignore` looks correct to me. A single line in this file should do the job:
```
Johnny Appleseed/NoSync/
```
This will exclude the folder "Johnny Appleseed/NoSync" and everything within it from uploading. Note that you will need to pause and resume syncing for any changes to `.mignore` to take effect.
